framework/src/controller/controller.js

```diff
@@ -117,7 +117,7 @@ class Controller {
 		const pidPath = `${this.config.dirs.pids}/controller.pid`;
 		const pidExists = await fs.pathExists(pidPath);
 		if (pidExists) {
-			const pid = parseInt(await fs.readFile(pidPath));
+			const pid = parseInt(await fs.readFile(pidPath), 10);
```

Comment:
I would prefer to use `Number(inputValue`) because it is easier to read. https://github.com/airbnb/javascript#coercion--numbers

Reply:
Im just standardising the eslint rule across the packages.
Also, https://eslint.org/docs/rules/radix has valid point

Reply:
I've checked the `radix` example and it is only useful for code written in pre-ECMAScript 5 which is not our case. We don't need the extra `, 10)`. In any case, I suggest to move to `Number` as Airbnb states is good too and it is less verbose.

Reply:
Im fine changing the rule, but let's change it across the packages in another issue